e2e/tests/oai/003-core-network.sh

```diff
@@ -49,24 +49,28 @@ function _wait_for_pfcp_session {
     debug "timeout: $timeout"
 }
 
+_core_kubeconfig="$(k8s_get_capi_kubeconfig "core")"
+_edge_kubeconfig="$(k8s_get_capi_kubeconfig "edge")"
+
 k8s_apply "$TESTDIR/003-core-network.yaml"
 
 for nf in nrf udm udr ausf amf smf; do
     k8s_wait_ready "packagevariant" "oai-$nf"
 done
 k8s_wait_ready "packagevariant" "oai-upf-edge"
 
-for nf in nrf udm udr ausf amf smf; do
-    kpt_wait_pkg "core" "oai-$nf" "nephio" "1800"
-done
-kpt_wait_pkg "edge" "oai-upf"
-
-_core_kubeconfig="$(k8s_get_capi_kubeconfig "core")"
-_edge_kubeconfig="$(k8s_get_capi_kubeconfig "edge")"
-for nf in nrf udm udr ausf amf smf; do
+for nf in nrf udm udr ausf amf; do
+    kpt_wait_pkg "core" "oai-$nf"
```

Comment:
@electrocucaracha Here you aren't specifying any logout you are just using the default one. Whereas in SMF you are. Don't we need to specify for AMF too?

Reply:
I just noticed that most of the timeouts that I was getting were caused by the dependency between SMF and UPF, so I decided to exclude it from the loop.